sdk-commons/src/main/java/org/openmrs/maven/plugins/model/DistroProperties.java

```diff
@@ -109,46 +109,59 @@ private String extractPropertyName(String key) {
         }
     }
 
```

Comment:
We can review this in more detail in SDK-348, but for now I have consolidated the two methods of specifying a parent distro so that both are equally supported.  See change below.